README.md

```diff
@@ -407,6 +407,19 @@ These settings can be changed at any time.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
 * `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
 
+#### Container image registry settings
+
+The following setting is optional and allows you to configure image registry mirrors and pull-through caches for your orchestrated containers.
+* `settings.image-registry.mirrors`: A mapping of image registry to a list of image registry URL endpoints.  When pulling an image from a registry, the container runtime will try the endpoints one by one and use the first working one.
+  (Docker and containerd will still try the default registry URL if the mirrors fail.)
+  * Example user data for setting up image registry mirrors:
+  ```
+  [settings.image-registry.mirrors]
+  "docker.io" = ["https://<my-docker-hub-mirror-host>"]
+  "gcr.io" = ["https://<my-gcr-mirror-host>","http://<my-gcr-mirror-host-2>"]
+  ```
```

Comment:
🚲 🏠 ⚠️

I saw you fixed this in the README , but `container-registry` might be a better term for the setting since "images" are kind of ambiguous.

In the context of a running Bottlerocket it's probably clear enough, but we deal with a lot of different image types - full disk images, filesystem images, etc.